Mail/Views/New Message/Attachments/Attachable.swift

```diff
@@ -38,27 +38,32 @@ extension NSItemProvider: Attachable {
         return UTType(preferredIdentifier)
     }
 
-    public func writeToTemporaryURL() async throws -> URL {
+    public func writeToTemporaryURL() async throws -> (url: URL, title: String?) {
```

Comment:
maybe a marginal improvement using a typealias for this `(url: URL, title: String?)`